packages/rest/test/unit/rest-application/rest-application.test.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  RestApplication,
+  RestServer,
+  RestComponent,
+  ERR_NO_MULTI_SERVER,
+} from '../../..';
+import {expect} from '@loopback/testlab';
+
+describe('RestApplication', () => {
+  describe('throws', () => {
+    it('when attempting to bind another server', () => {
+      const app = new RestApplication();
+      expect.throws(
+        () => {
+          app.server(RestServer, 'oops');
+        },
+        Error,
+        ERR_NO_MULTI_SERVER,
+      );
+    });
+
+    it('when attempting to bind an array of servers', () => {
+      const app = new RestApplication();
+      expect.throws(
+        () => {
+          app.servers([RestServer, RestServer]);
+        },
+        Error,
+        ERR_NO_MULTI_SERVER,
+      );
+    });
+
+    it('when attempting to bind servers via configuration', () => {
+      expect.throws(
+        () => {
+          const app = new RestApplication({
+            servers: {
+              foo: RestServer,
+              bar: RestServer,
+            },
+          });
+          // tslint complains about unused vars if app isn't used.
+          app.findByTag('server');
```

Comment:
Curious: if I don't call `app.findByTag`, will I create a rest app with multiple servers successfully? or I will receive an error? I feel the later one would be the expected behavior.

Reply:
I think @kjdelisle uses this line to work around a `tslint` style complaint. But as I commented, We should add `/* tslint:disable-next-line:no-unused-variable */` to make an exception.

Reply:
oh I see! was confused with the comment, now it makes sense.